src/test/java/seedu/address/logic/commands/SuggestCommandTest.java

```diff
@@ -98,7 +100,7 @@ public void equals() {
         assertFalse(suggestCommand.equals(null));
 
         // different predicate list -> returns false
```

Comment:
Should be updated since this isn't passing in `predicateList` anymore

Reply:
done